src/app/components/MeasurementPlot.tsx

```diff
@@ -104,36 +107,51 @@ function MeasurementComponent(props: MeasurementPlotProps) {
   });
   usePan({ horizontalAxisId: xAxis, verticalAxisId: yAxis });
   return (
-    <Plot width={width} height={height}>
-      <Heading title={title} />
-      <LineSeries
-        data={xyToXYObject({
-          x: x.data,
-          y: y.data,
-        })}
-        xAxis={xAxis}
-        yAxis={yAxis}
-      />
-      <Annotations>
-        {rectZoom.annotations}
-        {axisZoom.annotations}
-        {crossHairAnnot.annotations}
-      </Annotations>
-      <Axis
-        id={xAxis}
-        hidden={!showHorizontalAxis}
-        displayPrimaryGridLines={showVerticalGrid}
-        flip={flipHorizontalAxis}
-        position="bottom"
-        label={`${x.label}${x.units ? `(${x.units})` : ''}`}
-      />
-      <Axis
-        id={yAxis}
-        hidden={!showVerticalAxis}
-        displayPrimaryGridLines={showHorizontalGrid}
-        position="left"
-        label={`${y.label}${y.units ? `(${y.units})` : ''}`}
-      />
-    </Plot>
+    <div
+      css={css`
+        user-drag: none;
+        -webkit-user-drag: none;
+        user-select: none;
+        -moz-user-select: none;
+        -webkit-user-select: none;
+        -ms-user-select: none;
+      `}
+    >
```

Comment:
This div doesn't have a height. That's why the responsive chart takes 0px.

Reply:
fixed, is it expected that the plot keep stretching

Reply:
> is it expected that the plot keep stretching

no

Reply:
bug fixed